### PR TITLE
Fixing some includes 

### PR DESCRIPTION
**Pull request by jfamiglietti**
# Issue

When using Valhalla as a library in an iOS or Android app, these includes don't resolve correctly.

## Tasklist

 - [ ] Add tests
 - [ ] Add #fixes with the issue number that this PR addresses
 - [ ] Generally use squash merge to rebase and clean comments before merging
 - [ ] Update the [changelog](CHANGELOG.md)

## Requirements / Relations

 Link any requirements here. Other pull requests this PR is based on?

### CODE REVIEW

**Comment by mandeepsandhu** [comment]
Can you fix the linting errors causing the CI failures?

**Comment by jfamiglietti** [comment]
Updated. It took a while for one of them to complete, but it seems ok now.

File: src/mjolnir/statistics.h [hunk 1]
```diff
@@ -10,8 +10,8 @@
 
 #include <sqlite3.h>
 
-#include "baldr/graphconstants.h"
 #include "midgard/aabb2.h"
+#include "valhalla/baldr/graphconstants.h"
```
Comment (kevinkreiser) [inline src/mjolnir/statistics.h:14]: this one doesnt make sense to me. are you sure this diff is required?

**Review by kevinkreiser — CHANGES_REQUESTED** [review]
i think you can revert the change to statistics.h, or if that is required for you can you possibly show some compiler output so we can get a clue as to why its needed?

File: src/mjolnir/statistics.h (hunk 1 above)
Comment (jfamiglietti) [reply src/mjolnir/statistics.h]: Yeah, you're right. I just tested it and it's fine without the change. It doesn't end up in the include directory in the installed location anyway, so I wonder how I tripped over it. I pushed a change.

**Comment by kevinkreiser** [comment]
@jfamiglietti you'll want to actually revert that change, the formatter cares abotu what line that include lands on (it does some alphabetical sorting)

**Comment by jfamiglietti** [comment]
Yup, I noticed that. It's effectively reverted. Sorry for the noise.

**Review by kevinkreiser — APPROVED** [review]

**Pull request merged**